plugin.json

```diff
@@ -93,6 +93,12 @@
           "type": "text",
           "help_text": "Mattermost teams where sync is enabled (comma separated Mattermost team names, empty means all enabled)",
           "default": ""
+        },{
+          "key": "promptIntervalForDMsAndGMs",
+          "display_name": "Prompt interval for DMs and GMs (in hours)",
+          "type": "number",
```

Comment:
can we put the value as 0 here to disable the config, as the issue states regarding the disabling and enabling prompt